doc/parameters/param_list.md

```diff
@@ -176,6 +176,7 @@ When `plugins` parameter is not overridden, the following default plugins are lo
 | ----------| --------| ------------|
 | `<filter name>`.enabled | true | Whether it is enabled |
 | `<filter name>`.filter_info_topic | N/A | Name of the CostmapFilterInfo topic having filter-related information |
+| `<filter name>`.transform_tolerance | 0.0 | TF tolerance |
```

Comment:
We'll need a docs PR too to add this

Reply:
Yep. I suggest to separate these two PR in different repositories and process step-by-step on them.
I'd like to process with the documentation after we will agree about this PR and its design in order to make the documentation update over fixed version.

Reply:
Made https://github.com/ros-planning/navigation.ros.org/pull/109 PR on that

Reply:
That's fine, I was just letting you know & as a placeholder for me to look at so I don't merge this without forgetting